### ISSUE TITLE
Using env variable for setting up NEAR Social contract address

### ISSUE THREAD

**Issue by jaswinder6991** [near/devgigsboard#44]
At src/social_db.rs, we define social db contract's address as hardcoded:

```
use near_sdk::ext_contract;
use near_sdk::serde_json::Value;

pub const SOCIAL_DB: &str = "social.near";

#[ext_contract(ext_social_db)]
pub trait SocialDB {
    fn set(&mut self, data: Value);
}
```

While working on testnet and sandboxes, this have been a problem. I think we can use env variables to load the contract address.

**Comment by jaswinder6991** [comment]
I want to work on this.

**Comment by frol** [comment]
Indeed, it makes sense to be able to customize it for different networks.

There is no “env variables” concept in near contracts. You will need to store the value in the contract state and initialize on construction (`new` method)

**Comment by jaswinder6991** [comment]
Okay great. I will start working on including it in the contract state. I am assuming it would need migrations to be written too? I can take a crack at it as well.

**Comment by frol** [comment]
@jaswinder6991 I did not put too much thinking into it, but if you go the proposed route, you will need to implement the migration, indeed.
